src/views/Home/components/Banners/PerpetualBanner.tsx

```diff
@@ -0,0 +1,60 @@
+import { ArrowForwardIcon, Button, Text, useMatchBreakpoints, Link } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import Image from 'next/image'
+import { memo } from 'react'
+import styled from 'styled-components'
+import { perpetualImage, perpetualMobileImage } from './images'
+import * as S from './Styled'
+
+const RightWrapper = styled.div`
+  position: absolute;
+  right: 0;
+  bottom: 0px;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    bottom: 8.2px;
+  }
+  ${({ theme }) => theme.mediaQueries.md} {
+    bottom: 9px;
+  }
+  ${({ theme }) => theme.mediaQueries.lg} {
+    bottom: -2px;
+  }
+`
+const Header = styled(S.StyledHeading)`
+  font-size: 20px;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    font-size: 40px;
+  }
+`
+
+const PerpetualBanner = () => {
+  const { t } = useTranslation()
+  const { isDesktop } = useMatchBreakpoints()
+  return (
+    <S.Wrapper>
+      <S.Inner>
+        <S.LeftWrapper>
+          <S.StyledSubheading>{t('Perpetual Futures')}</S.StyledSubheading>
+          <Header width={['160px', '160px', 'auto']}>{t('Up to 100× Leverage')}</Header>
+          <Link href="https://perp.pancakeswap.finance/en/futures/BTCUSDT" external>
```

Comment:
Might be a good addition to add perpetualLangs check here

Reply:
@memoyil  Thanks.